src/d3a/setup/strategy_tests/external_devices.py

```diff
@@ -19,7 +19,8 @@
 from d3a.models.strategy.storage import StorageStrategy
 from d3a.models.strategy.load_hours import LoadHoursStrategy
 from d3a.models.strategy.external_strategies.pv import PVExternalStrategy
-from d3a.models.strategy.external_strategies.load import LoadHoursExternalStrategy
+from d3a.models.strategy.external_strategies.load import (LoadHoursExternalStrategy,
+                                                          LoadForecastExternalStrategy)
```

Comment:
Nit: imports can be optimized

Reply:
Not according to Pycharm, these are already optimized.

Reply:
Weird...pylint complains about them!

Reply:
> pylint: warning
> C0411 - third party import "from d3a_interface.constants_limits import ConstSettings" should be placed before "from d3a.models.area import Area" (wrong-import-order)

Reply:
Ah sorry! I misread the filename, you are of course right, fixing it.